app/components/sy-modal/header/template.hbs

```diff
@@ -1,6 +1,11 @@
 <div ...attributes class="modal-header">
   {{yield}}
-  <button type="button" class="close" aria-label="Close" {{on "click" @close}}>
+  <button
+    type="button"
+    class="close"
+    aria-label="Close"
+    {{on "click" (optional @close)}}
```

Comment:
Why is the close action optional?

Reply:
Artifact from previous refactoring attempt. Since the close action is wrapped before in a `optional` this should not be necessary anymore. Will delete.

Reply:
`done`